Sources/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.4.0</string>
+	<string>4.7.0</string>
```

Comment:
I think we are in version 4.6.0

Reply:
4.6.0 has already been released, so we're working on 4.7.0

Reply:
You never know, considering that Swift 5 may be out in not too long... We'll cross that bridge when we get to it 😄